code/java/search1.java

```diff
@@ -0,0 +1,70 @@
+package code.java;
+
+import org.xapian.Database;
+import org.xapian.Document;
+import org.xapian.Enquire;
+import org.xapian.MSet;
+import org.xapian.MSetIterator;
+import org.xapian.Query;
+import org.xapian.QueryParser;
+import org.xapian.Stem;
+
+public class search1 {
+    
+    // Command line args - dbpath querystring
+    public static void main(String[] args)
+    {
+        if(args.length < 2)
+        {
+            System.out.println("Insufficient number of arguments (should be dbpath querystring)");
+            return;
+        }
+        search(args[0], args[1]);
```

Comment:
You need to join all subsequent arguments together to make this compatible with the other programs. For instance, we run a multi-word search in `practical_examples/searching/running_the_search.rst` as `search1 db Dent watch`.